bin/i18n/curriculum_sync_utils.rb

```diff
@@ -43,14 +43,17 @@ def self.get_script_subdirectory(script)
   # default.
   def self.sync_in_serialize
     Script.all.each do |script|
-      # TODO: what else do we want to consider when deciding what to sync?
       next unless script.is_migrated?
+      next unless ScriptConstants.i18n? script.name
```

Comment:
do we still need to consider migrated here as well?

Reply:
unless we have a better way of distinguishing between scripts which do and do not have associated curriculum content!